resources/assets/js/core/components/typeahead.vue

```diff
@@ -81,17 +92,39 @@ export default {
             type: Object,
             default: () => TypeaheadItem,
         },
+        scrollable: {
+            type: Boolean,
+            default: false,
+        },
+        isLabelTree: {
+            type: Boolean,
+            default: false,
+        }
```

Comment:
Why do wen need `isLabelTree` here? The only difference I can see is that the typeahead item has height 44px then instead of 40px but why does it need another height?

Reply:
It has to do with the scrolling. If I don't set it to 44px for all items, the highlighted item can be hidden.

Reply:
You mean the scrolling by arrow keys? Can you maybe use the [clientHeight](https://developer.mozilla.org/en-US/docs/Web/API/Element/clientHeight) of the elements instead? It should be independent of the CSS if possible.

Reply:
If I use the clientHeight of the current highlighted element, the scrolling does not work properly any more because of different heights. I already tried different things, but this was the only solution that worked.